internal/release/promote/interactive_prompt_provider.go

```diff
@@ -61,11 +61,18 @@ func (i *InteractivePromptProvider) SelectReleases(list *cross.ReleaseList) (*cr
 	// Format releases for user selection.
 	var choices []string
 	for _, item := range list.Items {
-		inSync := item.AreVersionsInSync()
-		choice := fmt.Sprintf("%s\t%s\t>\t%s\t",
-			style.Resource(item.Name),
-			cross.GetReleaseDisplayVersion(item.Releases[1], inSync),
-			cross.GetReleaseDisplayVersion(item.Releases[0], inSync))
+		inSync := item.VersionInSync && item.ValuesInSync
+		var choice string
+		if inSync {
+			choice = fmt.Sprintf("%s\t%s",
+				style.PromotableRelease(item.Name, false),
+				GetReleaseDisplayVersion(item.Releases[1], true, true, true))
```

Comment:
Heated topic, but line 69 intention is now unknowable by directly reading it unless you have that function's context in your mind, the files are open side by side, or you use editor capabilities to inspect it.
I would strongly encourage to decouple content from style!

Reply:
Its ok! I find that `GetReleaseDisplayVersion(item.Releases[1], true, true, true)` is a little inscrutable. However if you like this, we can leave it as is!